src/eradiate/scenes/bsdfs/_hapke.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+import attrs
+import mitsuba as mi
+import pint
+import pinttr
+
+from ._core import BSDF
+from ..core import traverse
+from ..spectra import Spectrum, spectrum_factory
+from ... import validators
+from ...attrs import documented, parse_docs
+from ...kernel import TypeIdLookupStrategy, UpdateParameter
+from ...units import unit_context_config as ucc
+
+
+@parse_docs
+@attrs.define(eq=False, slots=False)
+class HapkeBSDF(BSDF):
+    """
+    Hapke BSDF [``rtls``].
+
+    This BSDF implements the Hapke surface model as described in
+    :cite:`Hapke1984BidirectionalReflectanceSpectroscopy`. This highly flexible
+    and robust surface model allows for the characterisation of a sharp
+    back-scattering hot spot. The so-called Hapke model has been adapted to
+    several different use cases in the litterature, the version with 6
+    parameters implemented here is one of the most commonly used.
+
+    See Also
+    --------
+    :ref:`plugin-bsdf-rpv`
```

Comment:
```suggestion
    :ref:`plugin-bsdf-hapke`
```
Check also if this link actually goes anywhere, I didn't verify the label in te plugin documentation.